AppSettings.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  ISetting,
+  SettingType,
+} from '@rocket.chat/apps-engine/definition/settings';
+
+export enum AppSettingId {
+  DialogflowBotUsername = 'dialogflow_bot_username',
```

Comment:
Why do you need the Dialogflow Bot Username?
This is not necessary. If this is part of the handoff process, then your App needs to provide an endpoint and then the Dialogflow chatbot needs to call it after the handoff. 
The endpoint will set the necessary information about the room object. You need to use the room object to store the information related to this integration (Salesforce Live Agent).

This app just can't have a dependency on the Dialogflow or any other App.